src/formatting_stack/strategies/impl/git_diff.clj

```diff
@@ -11,4 +11,4 @@
 
 ;; See: `git diff --help`, `diff-filter` section
 (speced/defn remove-markers [^string? s]
-  (-> s (string/replace-first #"^(A|C|D|M|R|T|U|X|B)\t" "")))
+  (-> s (string/replace-first #"^(A|C|D|M|R|T|U|X|B|\?|\!).*\t" "")))
```

Comment:
I guess the changes also target these:

![image](https://user-images.githubusercontent.com/1162994/75961186-ced53600-5ec1-11ea-84a4-fb6237d2a7d4.png)

Include them in unit tests pls

Reply:
Wasn't added because of the comment `;; `B` is replaced (and so on for other letters - we're not testing a whole regex range)`. will add anyway though.

Reply:
I see! At the same time, the requirements have changed hence the extra caution.